deploy/eck-elasticsearch/values.yaml

```diff
@@ -81,10 +129,355 @@ nodeSets:
+    #     - weight: 1
+    #       preference:
+    #         matchExpressions:
+    #         - key: another-node-label-key
+    #           operator: In
+    #           values:
+    #           - another-node-label-value
+    #   podAffinity:
+    #     requiredDuringSchedulingIgnoredDuringExecution:
+    #     - labelSelector:
+    #         matchExpressions:
+    #         - key: security
+    #           operator: In
+    #           values:
+    #           - S1
+    #       topologyKey: topology.kubernetes.io/zone
+    #   podAntiAffinity:
+    #     preferredDuringSchedulingIgnoredDuringExecution:
+    #     - weight: 100
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchLabels:
+    #             elasticsearch.k8s.elastic.co/cluster-name: quickstart
+    #         topologyKey: kubernetes.io/hostname
+
+    # AutomountServiceAccountToken indicates whether a service account token should be automatically mounted.
+    # automountServiceAccountToken: false
+
+    # Containers array.  Should only be used to customize the 'elasticsearch' container using the following fields.
       containers:
       - name: elasticsearch
+        # Arguments to the entrypoint of the 'elasticsearch' container.
+        # args: []
+
+        # Entrypoint array.  Not executed within a shell.
+        # command: []
+
+        # List of environment variables to set in the 'elasticsearch' container.
+        # env:
+        # name/value environment variable pair to set within the 'elasticsearch' container.
+        # - name: "my-env-var"
+        #   value: "my-value"
+        # Set a named environment variable's value from a configuration source.
+        # - name: "my-env-from"
+        #   valueFrom:
+            # Set the value from a configmap.
+            # configMapKeyRef:
+            #   name: specialEnvConfig
+            # Set the value from the container's field.
+            # fieldRef:
+            #   fieldPath: spec.nodeName
+            # Set the value from the container's resource requests.
+            # resourceFieldRef:
+            #   containerName: "elasticsearch"
+            #   resource: requests.cpu
+            # Set the value from a secret.
+            # secretKeyRef:
+            #   name: specialEnvSecretConfig
+            #   key: "my-key"
+
+        # List of sources to populate environment variables in the 'elasticsearch' container.
+        # envFrom:
+        # Set environment variables from key/value pairs within a ConfigMap.
+        # - configMapRef:
+        #     name: specialEnvConfig
+        # An optional identifier to prepend to each key in the ConfigMap.
+          # prefix: "DS_"
+        # Set environment variables from key/value pairs within a Secret
+        # - secretRef:
+        #     name: specialEnvSecretConfig
+            # Specify whether the Secret must be defined
+            # optional: false
+
+        # Container image name.
+        # image: ""
```

Comment:
Same here: while it is technically possible to set the image value in the PodTemplate directly, I would not advertise this as we have a top level field in the CRD that should be used for this.